### ISSUE TITLE
Member/Board Mutual Agreement

### ISSUE THREAD

**Issue by patrick727** [rchain/bounties#564]
Lets spend a week or two thinking about a mutual agreement to hold members/board accountable

https://docs.google.com/document/d/1x3j8fERfWDBxyJZij8MChOrhzazmRgspxgNNRUkDHMw/edit

this is a draft from Jon West based on the discord discussions.

**Comment by dckc** [comment]
@Patrick727: where are we on this issue? Suggestions for contacting Jon West?

I gather from @lapin7 in #616 that there's a board meeting tomorrow, May 1st. I've heard talk of the board sharing agendas with the membership in advance. Are we doing that yet?

@kennyrowe @leithaus @ian-bloom ?

**Comment by allancto** [comment]
@patrick727 , in reference to your first comment above) the draft from Jon West included input from several members. Obviously no contribution was huge (all measurable in minutes or hours), and in this there is no closure since the document is still unfinished and this issue isn't closed.Are these member's time and contribution considered valuable? If so, should they be compensated for their involvement? 

There is and should be an RChain cultural understanding that members should be encouraged to participate and those who do participate should and will be rewarded for their participation. This principle of encouraging participation of membership and rewarding accordingly has been stated on multiple occasions by our board @kennyrowe @leithaus @ian-bloom @lapin7 . It's also a requirement of our structure as a cooperative that members be encouraged to participate and rewarded for their participation (which i understand to have been the intent behind the recent resolution which is the subject of #616).

**Comment by allancto** [comment]
@dckc , you are correct in your understanding that a commitment was made to publishing board meeting agendas in advance, as well as publishing minutes in a timely fashion. There may have been hiccups in the past with this process, but radical transparency is a bedrock principle of the RChain Cooperative and we've been fixing hiccups as rapidly as we can.

**Comment by dckc** [comment]
@kennyrowe writes in discord:
> the agenda is typically not published ahead of time. Though that is something we could look into

**Comment by dckc** [comment]
a commitment was made to publishing board meeting agendas in advance? really? In what context was that commitment made, @allancto ? Is there some record of it? Got a pointer?

**Comment by dckc** [comment]
@allancto is there more to your question to @patrick727 about whether contributors should be compensated? It seems obvious that the answer is yes, they should, and our usual voting process is designed to determine how much.

> We consider it fair if the value you provide towards our goal is proportional to the value you receive.
 -- https://github.com/rchain/bounties/blob/master/README.md

**Comment by dckc** [comment]
> i'd ask in board<->membership if we had that channel

#board2coop is close enough, for my purposes. As noted above, @kennyrowe replied there:
> the agenda is typically not published ahead of time. Though that is something we could look into

I can check the video recording of the RAM meeting (#614). This is the 2nd time it's been suggested to me that it would be worth my time; i.e. that the written notes don't capture all the essential stuff or that some stuff that might usually be considered inessential is nonetheless worth my time.

**Comment by allancto** [comment]
@dckc , a bit more info, iirc correctly there was a concern expressed that some items might no be ready for public disclosure, so those aspects would be withheld (preferably temporarily, with a release date included for archival purposes).  Also I requested that in addition to the agenda, relevant items such as resolutions under consideration and ancillary materials also be provided, again subject to the needs of (temporary where possible) non disclosure. In general the board has indeed tried to be radically transparent, but occasionally the ball has been dropped because of glitches of one sort or another. As the saying goes, the spirit is willing but the process hasn't been implemented. -Allan

**Comment by dckc** [comment]
OK... I see in the Apr 25 RAM meeting notes:
> We also need some way to publish the Board agenda in advance so Members can suggest topics or provide feedback in advance

I also reviewed the video. There's clear momentum in the direction of sharing the board agenda in advance, but not what I'd call a commitment; nobody there was in a position to speak for the board.

But @ian-bloom , @lapin7 please do follow up on sharing board meeting agendas with the membership in advance.

**Comment by dckc** [comment]
Why suggest that I invite them? Why not do it yourself?

I prefer to make direct requests to one or a few people to avoid the
anybody/somebody/nobody anti-pattern.

Plus, only Ian and H J expressed support agendas in advance. I don't have
that info from other board members.

**Comment by dckc** [comment]
p.s. regarding...
> Also I requested that in addition to the agenda, relevant items such as resolutions under consideration and ancillary materials also be provided

I'm a web guy; I assume the agenda is one page but includes other linked materials by reference.

**Comment by barneycinnamon** [comment]
Might it be helpful to see if Nathan Schneider or his friend who knows about co-op governance could provide a few comments or model bylaws we could review for inspiration in how other co-ops' boards perform their service? _...heads over to Discord to ping Nathan_

**Comment by dckc** [comment]
@ian-bloom reports the coop is in the final stages of hiring someone to help with timely board minutes.

**Comment by dckc** [comment]
https://github.com/rchain/board is up-to-date now (c2e0d7f).

So... what's next? Is there any energy for continued work on this? Or is it **wontfix**?

**Comment by kennyrowe** [comment]
Good question. Since the last comment we hired Ashley to help with the
board organization, and I've started a weekly board Q&A. I could see how
some additional written documentation could be helpful, but perhaps it
would good to think again about what the goal here is?

**Comment by dckc** [comment]
I'm going to call this fixed; anybody with a better idea can re-open it (or make a new issue).

**Comment by allancto** [comment]
@dckc @kennyrowe i agree this should be closed. The original impetus had to do with moderation of the discord channel, and over time and personal dm's our culture seems to have evolved to the point where there's a lot more mutual respect in the channels. Kenny's weekly office hours have gone a long way to unshrouding the mystery of what happens in the board meetings, @deannald has also helped in opening communication channels around specific topics. I expect that the runup to the upcoming annual meeting, proposals and campaigning (and eventually additional person-hours on the board) will also result in the kind of transparency which I think everyone has always wanted and has is happening a bit slowly but very surely.

**Comment by deannald** [comment]
@allancto Thanks for your input. That reminds me, I need to update the membership on Discord regarding the email on the annual meeting - Ian hit a legal snag on wording, but the email should be out soon...

**Comment by dckc** [comment]
@kennyrowe Hiring Ashly doesn't seem to have fixed anything. We're back to having the most recent board minutes 2.5 months old:

https://github.com/rchain/board/tree/master/2018
